site/mkdocs.yml

```diff
@@ -11,7 +11,12 @@ nav:
 - About: index.md
 - Get Started:
   - get-started/quickstart.md
-  - samples/notebook.md
+  - Notebook samples:
+    - samples/tools.md
```

Comment:
Perhaps this should be under an intro / before we begin?

Reply:
I moved it under before we begin page